Controller/Payment/Checkout/Token.php

```diff
@@ -122,4 +141,28 @@ public function execute()
             ->setHttpResponseCode(\Magento\Framework\Webapi\Response::HTTP_OK)
             ->setData($resBody);
     }
+
+    /**
+     * CreateBuyer
+     *
+     * @param $transactionId
+     */
+    public function createBuyer($transactionId)
+    {
+        try {
+            if ($this->customerSession->isLoggedIn() && empty($this->helper->getBuyerId()) && $transactionId) {
+                $response = $this->requestFactory
+                    ->create(RequestFactory::TRANSACTIONS_REQUEST_METHOD)
+                    ->setRequestMethod('transactions/' . $transactionId)
+                    ->setTopic('gettransactionid')
```

Comment:
What is this doing?

Reply:
This calls the GET request for transaction with transactionID.